components/brave_ads/browser/ads_notification_handler.cc

```diff
@@ -120,4 +157,47 @@ const void* AdsNotificationHandler::UserDataKey() {
   return &kUserDataKey;
 }
 
+void AdsNotificationHandler::OperationCompleted(
+    const std::string& notification_id) {
+
+  auto iter = pending_callbacks_.find(notification_id);
+  std::move(iter->second).Run();
+  pending_callbacks_.erase(iter);
+#if defined(OS_ANDROID)
+  StartShutDownTimerIfNecessary(notification_id);
+#endif
+}
+
+#if defined(OS_ANDROID)
+bool AdsNotificationHandler::IsHeadless(){
+  auto state = base::android::ApplicationStatusListener::GetState();
+  bool uiless = (state ==
```

Comment:
Nit: Seems like we use `headless`, `no_ui`, and `uiless`. I would just stick with one everywhere for consistency so it's clear that they all refer to the same thing.

Reply:
Using `headless` everywhere